k8t/templates.py

```diff
@@ -49,7 +53,7 @@ def analyze(template_path: str, values: dict, engine: Environment) -> Tuple[Set[
         var for var in required_variables if var in PROHIBITED_VARIABLE_NAMES
     }
 
-    return (undefined_variables - invalid_variables), unused_variables, invalid_variables, secrets
+    return (undefined_variables - invalid_variables), unused_variables, invalid_variables, has_secrets
 
 
 def validate(template_path: str, values: dict, engine: Environment) -> bool:
```

Comment:
Probably let's rename `secrets` -> `has_secrets` here as well to make it more obvious that it's  boolean?